### PR TITLE
Add instructions about crontab in installation guide

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

File: developer_guide/installation/installation_workstation.rst [hunk 1]
```diff
@@ -375,6 +375,33 @@ Adding the vhost name
     $ sudo gedit /etc/hosts
     127.0.0.1    akeneo-pim.local
 
+Configuring tasks via crontab
+*****************************
+
+The application needs the following tasks to be executed in background on a regular basis :
```
Comment (rybus) [inline developer_guide/installation/installation_workstation.rst]: no space between a word and :

File: developer_guide/installation/installation_workstation.rst [hunk 2]
```diff
@@ -375,6 +375,33 @@ Adding the vhost name
...
+    # for EE only
+    /path/to/php /path/to/pim/root/app/console akeneo:rule:run
+
+Edit your crontab with `crontab -e` and configure each task. For example, the following line will run the completeness calculation every 15 minutes :
+
+.. code-block:: bash
+    :linenos:
+
+    # m  h  dom  mon  dow  command
+    */15 *  *    *    *    /path/to/php /path/to/pim/root/app/console pim:completeness:calculate > /path/to/pim/root/app/logs/calculate_completeness.log 2>&1
```
Comment (rybus) [inline developer_guide/installation/installation_workstation.rst]: Maybe add a note about 2>&1 to say it redirect stderr to stdout wich is the log file

**Comment by rybus** [comment]
Small notices but GTM

File: developer_guide/installation/installation_workstation.rst [hunk 3]
```diff
@@ -375,6 +375,33 @@ Adding the vhost name
...
+Configuring tasks via crontab
+*****************************
+
+The application needs the following tasks to be executed in background on a regular basis :
+
+.. code-block:: bash
+    :linenos:
+
+    # for CE & EE
+    /path/to/php /path/to/pim/root/app/console pim:completeness:calculate
```
Comment (skeleton) [inline developer_guide/installation/installation_workstation.rst]: add `--env=prod`

Comment (SamirBoulil) [inline developer_guide/installation/installation_workstation.rst]: is '/path/to' really necessary ? I mean we "get" it right ?  I mean it doesn't help to read that's all.

**Comment by skeleton** [comment]
gtm once fixed :+1:

**Comment by SamirBoulil** [comment]
GTM

**Pull request merged**